Automattic/wp-calypso
# Hosting Command Palette: Display notice for filtered sites

Fixes https://github.com/Automattic/dotcom-forge/issues/4827

## Proposed Changes

Displays a notice when the site list is filtered:

![CleanShot 2023-12-15 at 15 11 07@2x](https://github.com/Automattic/wp-calypso/assets/36432/009cc011-e059-46a4-910b-62e343d36fa0)


![CleanShot 2023-12-15 at 15 10 54@2x](https://github.com/Automattic/wp-calypso/assets/36432/eeb57983-52cf-4887-94a5-7ef0d0f785f7)


I used 'Only listing sites with hosting features enabled.' for any hosting-adjacent features.

## Testing Instructions

1. Review all of the strings and verify they appear as expected.
2. Test out the commands and verify the notice appears as expected.
3. Test out other commands and verify the notice doesn't appear.
